CONTRIBUTING.md

```diff
@@ -10,6 +10,8 @@ Follow the golder rule:
 
 The first thing to do always is to install the extension and note how it works. One of the best ways to do this is to launch the application and go through the files that it launches sequentially.
 
+NOTE - While testing, to avoid confusion the original extension can be disabled. But however, both of them can work at the same time.
```

Comment:
```suggestion
NOTE - For testing the development version, it is recommended to disable the original extension as it might cause confusion and log into the ERP even when development version might not be working properly.
```